File: RailwayWizzard.Bot/bot/data/CarType.py
from enum import Enum


class CarType(Enum):
    SEDENTARY = ('Сидячий', True)
    SEDENTARY_BUSINESS = ('Сидячий бизнес', False)
    RESERVED_SEAT_UPPER = ('Плац верх', True)
    RESERVED_SEAT_LOWER = ('Плац низ', True)
    RESERVED_SEAT_UPPER_SIDE = ('Плац верх бок ', False)
    RESERVED_SEAT_LOWER_SIDE = ('Плац низ бок', False)
    COMPARTMENT_UPPER = ('Купе верх', True)
    COMPARTMENT_LOWER = ('Купе низ', True)
    LUXURY = ('СВ', False)

    @property
    def display_name(self):
        return self.value[0]

    @property
    def is_sitting(self):
        return self.value[1]

File: RailwayWizzard.Core/NotificationTask/CarType.cs
using System.ComponentModel;

namespace RailwayWizzard.Core.NotificationTask;

/// <summary>
/// Класс обслуживания.
/// </summary>
public enum CarType
{
    [Description("Сидячий")]
    Sedentary = 1,

    [Description("Сидячий (бизнес)")]
    SedentaryBusiness = 2,

    [Description("Плацкарт верхнее")]
    ReservedSeatUpper = 3,

    [Description("Плацкарт нижнее")]
    ReservedSeatLower = 4,

    [Description("Плацкарт верхнее боковое")]
    ReservedSeatUpperSide = 5,

    [Description("Плацкарт нижнее боковое")]
    ReservedSeatLowerSide = 6,

    [Description("Купе верхнее")]
    CompartmentUpper = 7,

    [Description("Купе нижнее")]
    CompartmentLower = 8,

    [Description("СВ")]
    Luxury = 9
}